IBX-2362: Added dynamic targeting block (#124)
ezsystems/user-documentation: docs/site_organization/working_with_page.md

|Block|Description|
|-----|-----------|

> |Targeting|Embeds an Content item depending on the [Segment](organizing_the_content.md#segments) the user belongs to. For details, see [Targeting block](#targeting-block).|
> |Dynamic targeting|Embeds recommended items depending on the [Segment](organizing_the_content.md#segments) the user belongs to. For details, see [Dynamic targeting](#dynamic-targeting).|

Reviewer: ```suggestion
|Targeting|Embeds Content items based on the [Segment](organizing_the_content.md#segments) the user belongs to. For details, see [Targeting block](#targeting-block).|
|Dynamic Targeting|Embeds recommended items based on the [Segment](organizing_the_content.md#segments) the user belongs to. For details, see [Dynamic Targeting block](#dynamic-targeting-block).|
```

### Targeting block

> Dynamic targeting block provides recommendation of content based on users related to the configured Segments.

Reviewer: ```suggestion
Targeting block provides Content item recommendations based on user groups that are related to selected Segments.
```

![Previewing Page for a given Segment](img/page_builder_segment_preview.png)

> ### Dynamic targeting

Author: The label is as on the UI: Dynamic targeting

> Dynamic targeting block provides recommendation items based on users related to the configured Segments.

Reviewer: ```suggestion
Dynamic targeting block provides recommended items based on scenarios that apply to users related to the configured Segments.
```

On the **Basic** tab, perform the following actions:

- In the **Name** field, provide the name of the block.

> - In the **Select default scenario** section, select the default scenario for recommendation items that should be rendered if the current user

Reviewer: ```suggestion
- In the **Select default scenario** section, select the default scenario to generate a list of recommended items that should be rendered 
```

> is not assigned to any Segments.

Reviewer: ```suggestion
if the current user is not assigned to any Segment.
```

- In the **Setup segment and scenario matching priority rules** drop-down, select a Segment group, a Segment identifier and Scenario that you want to display recommendations from.

> The rules are checked in order, so if a user belongs to more than one Segment, the first rule applies.

Reviewer: ```suggestion
The rules are checked in order, so when a user belongs to more than one Segment, the first rule applies.
```
